docs/blog/e18e.md

```diff
@@ -31,21 +31,21 @@ e18e is a community initiative to bring together people who are passionate about
 
 Our focus is on three main areas:
 
-- **cleanup** - reducing, simplifying and modernizing dependency trees.
-- **speedup** - speeding up core packages and frameworks.
-- **levelup** - providing modern, lighter alternatives to core packages.
+- **Clean up**: reducing, simplifying and modernizing dependency trees
+- **Speed up**: speeding up core packages and frameworks
+- **Level up**: providing modern, lighter alternatives to core packages
```

Comment:
similar as the ones before, these are names. they are fine to remain as they were (but with the colons)